Fixes:
implicit declaration of function `strdup'
implicit declaration of function `strcasecmp'

$OpenBSD: patch-gphoto2_actions_c,v 1.1 2008/12/16 08:48:08 ajacoutot Exp $
--- gphoto2/actions.c.orig	Tue Dec 16 09:01:12 2008
+++ gphoto2/actions.c	Tue Dec 16 09:01:55 2008
@@ -18,7 +18,9 @@
  * Boston, MA 02111-1307, USA.
  */
 
+#ifndef __OpenBSD__
 #define _XOPEN_SOURCE	/* strptime proto, but this hides other prototypes */
+#endif
 #define _GNU_SOURCE	/* get all the other prototypes */
 #define __EXTENSIONS__	/* for solaris to get back strdup and strcasecmp */
 
